YDotNet/YDotNetException.cs

```diff
@@ -33,14 +32,4 @@ public YDotNetException(string message, Exception inner)
         : base(message, inner)
     {
     }
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="YDotNetException"/> class for serialization.
-    /// </summary>
-    /// <param name="info">The serialization info.</param>
-    /// <param name="context">The serialization context.</param>
-    protected YDotNetException(SerializationInfo info, StreamingContext context)
```

Comment:
@SebastianStehle Is this needed to migrate to .NET 8?

Reply:
BinaryFormatter is obsolete now.